ivy/functional/frontends/paddle/stat.py

```diff
@@ -52,6 +52,15 @@ def numel(x, name=None):
     return ivy.array(prod if prod > 0 else ivy.array(length, dtype=ivy.int64))
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def quantile(x, q, axis=None, keepdim=False):
+    return ivy.quantile(x, q, axis=axis, keepdims=keepdim)
```

Comment:
Paddle quantile returns float64 to obtain higher precision.

Reply:
So should I mention only float64?

Reply:
AssertionError: returned dtype = float32, ground-truth returned dtype = float64

Reply:
@Sarvesh-Kesharwani This has to be implemented within the Test file or this one?

Reply:
@Sarvesh-Kesharwani I have added. Can you check?